addons/website_slides_survey/tests/test_course_certification_stats.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo.addons.survey.tests.common import TestSurveyCommon
+
+
+class TestCourseCertificationStats(TestSurveyCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        # Create certifications
+        cls.certifications = cls.env["survey.survey"].create(
+            [
+                {
+                    "title": f"certification {i + 1}",
+                    "scoring_type": "scoring_with_answers",
+                    "certification": True,
+                }
+                for i in range(3)
+            ]
+        )
+
+        # Create courses and link them to certifications
+        [cls.course_1, cls.course_2, cls.course_3] = cls.env["slide.channel"].create(
+            [
+                {
+                    "name": f"Course {i +1}",
+                    "enroll": "public",
+                    "is_published": True,
+                    "slide_ids": [Command.create(
+                        {
+                            "name": f"Certification {i + 1} slide",
+                            "slide_category": "certification",
+                            "survey_id": cls.certifications.ids[i],
+                            "is_published": True,
+                        })]
+                }
+                for i in range(3)
+            ]
+        )
+
+        # Create course participants
+        cls.participants = cls.env["res.users"].browse([
+            cls.survey_user.id,
+            cls.user_emp.id,
+            cls.user_portal.id
+        ])
```

Comment:
Sorry, my previous syntax was wrong, it's actually:

`cls.participants = cls.survey_user + cls.user_emp + cls.user_portal`
(The `|` symbol also works when you want to avoid duplication, but it's not necessary here).